actions/profile/edit.php

```diff
@@ -51,8 +51,8 @@ function profile_array_decoder(&$v) {
 		forward(REFERER);
 	}
 
-	if ($value && $valuetype == 'url' && !preg_match('~^https?\://~i', $value)) {
-		$value = "http://$value";
+	if ($value && $valuetype == 'url') {
+		$value = elgg_normalize_url($value);
```

Comment:
I don't think we want this. elgg_normalize_url() will passthrough URLs like "javascript:alert(1)".